RELEASING.md

```diff
@@ -1,48 +1,63 @@
+    e.g. `release/v1.9.x`, and click the "Run workflow" button below that.
+* Review and merge the pull request that it creates.
+
+## Making the release
+
+Run the [Release workflow](https://github.com/trask/repository-template/actions/workflows/release.yml).
+
+* Press the "Run workflow" button, then select the release branch from the dropdown list,
+  e.g. `release/v1.9.x`, and click the "Run workflow" button below that.
+* This workflow will publish the artifacts and publish a GitHub release with release notes based on the change log.
+* Review and merge the pull request that the release workflow creates against the release branch
+  which adds the release date to the change log.
+
+## After the release
+
+Run the [Merge change log to main workflow](https://github.com/trask/repository-template/actions/workflows/merge-change-log-to-main.yml).
+
+* Press the "Run workflow" button, then select the release branch from the dropdown list,
+  e.g. `release/v1.9.x`, and click the "Run workflow" button below that.
+* This will create a pull request that merges the change log updates from the release branch
+  back to `main`.
+* Review and merge the pull request that it creates.
+* This workflow will fail if there have been conflicting change log updates introduced in `main`,
+  in which case you will need to merge the change log updates manually and send your own pull
+  request against `main`.
+
 
 Release Process:
-* [Checkout a clean repo](#checkout-a-clean-repo)
-* [Update versions](#update-versions)
-* [Create a new branch](#create-a-new-branch)
-* [Open a Pull Request](#open-a-pull-request)
-* [Create a Release](#Create-a-Release)
+* [Check PyPI](#Check-PyPI)
 * [Move stable tag](#Move-stable-tag)
 * [Update main](#Update-main)
-* [Update docs](#Update-docs)
-* [Check PyPI](#Check-PyPI)
 * [Troubleshooting](#troubleshooting)
 
-## Checkout a clean repo
-To avoid pushing untracked changes, check out the repo in a new dir
-
-## Update versions
-The update of the version information relies on the information in eachdist.ini to identify which packages are stable, prerelease or
-experimental. Update the desired version there to begin the release process.
-
-## Create a new branch
-The following script does the following:
-- update main locally
-- creates a new release branch `release/<version>`
-- updates version and changelog files
-- commits the change
-
-*NOTE: This script was run by a GitHub Action but required the Action bot to be excluded from the CLA check, which it currently is not.*
-
-```bash
-./scripts/prepare_release.sh
-```
-
-## Open a Pull Request
-
-The PR should be opened from the `release/<version>` branch created as part of running `prepare_release.sh` in the steps above.
-
-## Create a Release
-
-- Create the GH release from the main branch, using a new tag for this micro version, e.g. `v0.7.0`
-- Copy the changelogs from all packages that changed into the release notes (and reformat to remove hard line wraps)
-
```

Comment:
Perhaps we can add a TODO to automate the updating `stable` tag below as well?

Reply:
The `stable` tag is automatically bumped as part of the workflow already (unless I misunderstood your question)

Reply:
Could you link to the line in which this happens? I can't seem to find it.

Reply:
ya, the `prepare-release-branch.yml` and `prepare-patch-release.yml` both call the `update-versions.sh` script, e.g. https://github.com/open-telemetry/opentelemetry-python/pull/2649/files#diff-69e7d09297eb3770a53d0e2385c3593d60acc3c787222c3809f6a540c01d498bR35

Reply:
This would be referring to updating the latest [stable](https://github.com/open-telemetry/opentelemetry-python/releases/tag/stable) tag to the release commit right? We need this for readthedocs to render the latest document changes.